tooling/fixtures/fixture_data/bls12-381-sha-256/signature/signature001.json

```diff
@@ -8,7 +8,7 @@
   "messages": [
     "9872ad089e452c7b6e283dfac2a80d58e8d0ff71cc4d5e310a1debdda4a45f02"
```

Comment:
How are these messages encoded, @BasileiosKal? Are these scalars (in hex), or octet strings that need to be encoded with MapMessageToScalar? When I try to parse these directly, I get an error because these are bigger than the order `r`.

Reply:
Looking at @tmarkovski's implementation, seems like you indeed need to first map the messages to scalar. This is a bit confusing because the spec's Sign and Verify function's `messages` param, matching the name in the fixture files, is of type scalar.

Reply:
@christianpaquin agreed I think clarifying text here about the form of messages in these test vectors is important. Also think the additional test case you suggest is a good one, there are few possible ones we could add.

Reply:
Have captured a note to follow up on this in #7